Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/Follow.cs

```diff
@@ -0,0 +1,644 @@
+        /// </summary>
+        public bool FaceTrackedObjectWhileClamped
+        {
+            get { return faceTrackedObjectWhileClamped; }
+            set { faceTrackedObjectWhileClamped = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("Face a user defined transform rather than using the solver orientation type.")]
+        private bool faceUserDefinedTargetTransform = false;
+
+        /// <summary>
+        /// Face a user defined transform rather than using the solver orientation type.
+        /// </summary>
+        public bool FaceUserDefinedTargetTransform
+        {
+            get { return faceUserDefinedTargetTransform; }
+            set { faceUserDefinedTargetTransform = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("Transform this object should face rather than using the solver orientation type.")]
+        private Transform targetToFace = null;
+
+        /// <summary>
+        /// Transform this object should face rather than using the solver orientation type.
+        /// </summary>
+        public Transform TargetToFace
+        {
+            get { return targetToFace; }
+            set { targetToFace = value; }
+        }
+
+        [SerializeField]
+        [EnumFlags]
+        [Tooltip("Rotation axes used when facing target.")]
+        private AxisFlags pivotAxis = AxisFlags.XAxis | AxisFlags.YAxis | AxisFlags.ZAxis;
+
+        /// <summary>
+        /// Rotation axes used when facing target.
+        /// </summary>
+        public AxisFlags PivotAxis
+        {
+            get { return pivotAxis; }
+            set { pivotAxis = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("Min distance from eye to position element around, i.e. the sphere radius")]
+        private float minDistance = 1f;
+
+        /// <summary>
+        /// Min distance from eye to position element around, i.e. the sphere radius.
+        /// </summary>
+        public float MinDistance
+        {
+            get { return minDistance; }
+            set { minDistance = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("Max distance from eye to element")]
+        private float maxDistance = 2f;
+
+        /// <summary>
+        /// Max distance from eye to element.
+        /// </summary>
+        public float MaxDistance
+        {
+            get { return maxDistance; }
+            set { maxDistance = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("Min distance from eye to position element around, i.e. the sphere radius")]
+        private float defaultDistance = 1f;
+
+        /// <summary>
+        /// Initial placement distance. Should be between min and max.
+        /// </summary>
+        public float DefaultDistance
+        {
+            get { return defaultDistance; }
+            set { defaultDistance = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("The element will stay at least this close to the center of view")]
```

Comment:
If the element says at least this close, shouldn't the variable be min as opposed to max?

Or should the tooltip (and comment below) read "no further than"?

Reply:
I spent way to long just now trying to work out this double negative, haha. I would say, the least close thing, is the furthest thing away. So "at least this close" means the same as "no further than this" (at least I think it does ¯\_(ツ)_/¯).

I think this is worth changing anyway, haha, it took me long enough to justify the wording so it could definitely be clearer.

Reply:
> So "at least this close" means the same as "no further than this"

Gotcha.... sorry if i was confusing, it's not only early in the year, but early in the _decade_! 😄